curriculum/challenges/english/08-coding-interview-prep/rosetta-code/align-columns.english.md

```diff
@@ -41,7 +41,7 @@ or$center$justified$within$its$column.
 ```yml
 tests:
   - text: <code>formatText</code> is a function.
-    testString: assert(typeof formatText === 'function', '<code>formatText</code> is a function.');
+    testString: assert(typeof formatText === 'function');
   - text: '<code>formatText</code> with the above input and "right" justification should produce the following: '
     testString: 'assert.strictEqual(formatText(testInput, ''right''), rightAligned, ''<code>formatText</code> with the above input and "right" justification should produce the following: '');'
```

Comment:
There are a few message arguments in `testString` that needs to be removed in this file.

Reply:
Fixed in separate commit.